test/unit/migrate_config_proxy_to_http_proxy.rb

```diff
@@ -0,0 +1,167 @@
+require 'test_plugin_helper'
+require File.join(Pathname.new(File.dirname(__FILE__)).parent.parent, 'db', 'migrate', '20200728054557_refer_config_proxy_to_http_proxy.rb')
```

Comment:
I think you can clean this up: `ForemanVirtWhoConfigure::Engine.root.join('db', ...)`